embedding/embedding.cpp

```diff
@@ -350,6 +351,7 @@ componentize_initialize() {
     Runtime.debug = true;
   }
 
+  builtins::web::performance::Performance::timeOrigin = std::chrono::steady_clock::now();
```

Comment:
Instead of in initialize, this should be done on the call to the first export, ie in the `__attribute__((export_name("call"))) uint32_t call` function, but with a check that it has not already been done.